api/conf/settings/misc.py

```diff
@@ -3,6 +3,8 @@
 from decouple import config
 
 
+ENABLE_COLLECTIONS = config("ENABLE_COLLECTIONS", cast=bool, default=False)
```

Comment:
Just a nit, I think there's a way to clarify this, but not 100% sure what the goal is. The name "enable" made me think at first that it was necessary to set this to true for the parameters to work at all. Is that the case? Is it more like "stabilise_collections" or something along those lines? Enable reads more binary on/off to me, and it would be surprising to me if I hadn't read this PR to see the parameters working when I request them in production but this set to false.

Reply:
Maybe `SHOW_COLLECTION_DOCS` better relays what this variable does?

In this PR, the parameters work correctly. This variable only affects the documentation. When set to False, the old documentation is shown (with the `/tag`, `/source` and `/source/sourceName/creator/` endpoints removed). When set to True, we can test the updated documentation for `creator`, `tag` and `collection` parameters.

Stabilizing would mean removing the `unstable__` prefix from the parameters, so it's not what the variable does.

Reply:
Gotcha! The show docs version you suggest would be great.

Reply:
@obulat Will you implement this change to the environment variable?